continuum-auth/src/jwt.test.ts

```diff
@@ -0,0 +1,49 @@
+import { verify } from 'jsonwebtoken';
+import { decodeJournalToken } from './jwt';
+import { None } from 'funfix';
+import { InfraLogger as logger } from './logger';
+
+jest.mock('jsonwebtoken');
+jest.mock('./logger');
+jest.mock('./config', () => ({
+    default: {
+        journal_jwt: {
+            secret: 'secret',
+        },
+    },
+}));
+
+describe('jwt', () => {
+    describe('decodeJournalToken', () => {
+        beforeEach(() => jest.resetAllMocks());
```

Comment:
You can do:
```
beforeEach(jest.resetAllMocks);
```

Reply:
No longer needed now that log checks have been removed from these tests.